File: Algorithms/944.Delete-Columns-to-Make-Sorted/delete_columns_to_make_sorted.go
func minDeletionSize(A []string) int {
	if len(A) == 0 {
		return 0
	}
	length := len(A[0])
	res := 0
	for j := 0; j < length; j++ {
		for i := 1; i < len(A); i++ {
			if A[i][j] < A[i-1][j] {
				res++
				break
			}
		}
	}
	return res
}

File: Algorithms/944.Delete-Columns-to-Make-Sorted/delete_columns_to_make_sorted.py
# -*- coding: utf-8 -*-

class Solution:
    def minDeletionSize(self, A: List[str]) -> int:
        res = 0
        length = len(A[0])
        for i in range(length):
            col = [ele[i] for ele in A]
            if col != sorted(col):
                res +=1
        return res
